pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/SnapshotSegmentAbortedTxnProcessorImpl.java

```diff
@@ -442,11 +443,13 @@ public class PersistentWorker {
         private final PersistentTopic topic;
 
         //Persistent snapshot segment and index at the single thread.
-        private final CompletableFuture<SystemTopicClient.Writer<TransactionBufferSnapshotSegment>>
+        private final ReferenceCountedWriter<TransactionBufferSnapshotSegment>
                 snapshotSegmentsWriterFuture;
```

Comment:
Should we rename the variable `snapshotSegmentsWriterFuture` to `snapshotSegmentsWriter` ?

Same for the variable `snapshotIndexWriterFuture`.

Reply:
fixed